packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -248,6 +248,22 @@ const parseBootOrder = bootOrder => {
   return order
 }
 
+const WarningIllegalSr = connectStore({
+  hosts: createGetObjectsOfType('host'),
+})(({ hosts, requiredHost }) => {
+  const host = hosts[requiredHost]
+  return (
+    <span className='text-danger'>
+      <Icon icon='alarm' />{' '}
+      {host !== undefined
+        ? _('srNotOnSameHost', {
+            host: host.name_label,
+          })
```

Comment:
As I'm seeing we're adding a 3rd message that says almost the same thing, I think we can simplify it and only have a generic message that only explains the issue without the host name. 'The SRs must either be shared or on the same host for the VM to be able to start.' seems pretty fine. So `WarningIllegalSr` would be a very simple component used 3 times eventually. Sorry if I said otherwise earlier, I don't remember.